Dockerfile

```diff
@@ -28,13 +28,18 @@ ENV ACTIVEMQ_HOME="/opt/activemq"
 ENV ACTIVEMQ_BASE="/opt/activemq"
 ENV ACTIVEMQ_CONF="/opt/activemq/conf"
 ENV ACTIVEMQ_DATA="/opt/activemq/data"
+ENV ACTIVEMQ_BROKER_NAME="localhost"
+
 
 ENV DOWNLOAD_URL="https://archive.apache.org/dist/activemq/${ACTIVEMQ_VERSION}/apache-activemq-${ACTIVEMQ_VERSION}-bin.tar.gz"
 ENV DOWNLOAD_ASC_URL="${DOWNLOAD_URL}.asc"
 ENV DOWNLOAD_KEYS_URL="https://downloads.apache.org/activemq/KEYS"
 
 ENV LC_ALL=C
 
+# ------------------------------------------------
+# Install ActiveMQ
+# ------------------------------------------------
 RUN mkdir -p ${ACTIVEMQ_HOME} /data /var/log/activemq && \
```

Comment:
Downloading and crafting the xml config should really be part of a separate build stage.
Please take a look at how bakery builds tomcat https://github.com/Alfresco/alfresco-dockerfiles-bakery/blob/117ef5872eb81ad604c52bbefb965f521761beb6/tomcat/Dockerfile#L50